ECIPs/ECIP-1049.md

```diff
@@ -0,0 +1,52 @@
+### What is an ECIP?
+
+    ECIP: 1049
+    Title: Change the ETC Proof of Work Algorithm to Keccak256
+    Status: Draft
+    Type: Meta
+    Author: Alexander Tsankov (alexander.tsankov@colorado.edu)
+    Created: 2019-01-08
+
```

Comment:
please add a `discussion-to` field with link to an issue or forum thread

Reply:
Can you open it as an issue as well, when i merge this the pr discussion gets buried .